[

{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#CVE-2020-15206", "@type": ["http://www.w3.org/2002/07/owl#NamedIndividual", "http://docs.oasis-open.org/cti/ns/stix#Vulnerability"], "http://docs.oasis-open.org/cti/ns/stix/core#description": [{"@value": "In Tensorflow before versions 1.15.4, 2.0.3, 2.1.2, 2.2.1 and 2.3.1, changing the TensorFlow's `SavedModel` protocol buffer and altering the name of required keys results in segfaults and data corruption while loading the model. This can cause a denial of service in products using `tensorflow-serving` or other inference-as-a-service installments. Fixed were added in commits f760f88b4267d981e13f4b302c437ae800445968 and fcfef195637c6e365577829c4d67681695956e7d (both going into TensorFlow 2.2.0 and 2.3.0 but not yet backported to earlier versions). However, this was not enough, as #41097 reports a different failure mode. The issue is patched in commit adf095206f25471e864a8e63a0f1caef53a0e3a6, and is released in TensorFlow versions 1.15.4, 2.0.3, 2.1.2, 2.2.1, or 2.3.1."}], "http://docs.oasis-open.org/cti/ns/stix/core#modified": [{"@value": "2021-03-13T20:12:26.98135Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#created": [{"@value": "2021-03-13T20:12:26.98135Z"}], "http://docs.oasis-open.org/cti/ns/stix/core#spec_version": [{"@value": "2.1"}], "http://docs.oasis-open.org/cti/ns/stix/core#id": [{"@value": "vulnerability--c428a5fd-2c66-476e-8f71-498328a8c05e"}], "http://docs.oasis-open.org/cti/ns/stix/core#name": [{"@value": "CVE-2020-15206"}], "http://docs.oasis-open.org/cti/ns/stix/core#created_by_ref": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a"}], "http://docs.oasis-open.org/cti/ns/stix/marking#object_marking_refs": [{"@id": "http://docs.oasis-open.org/tac/ns/stix-cve#marking-definition--62fd3f9b-15f3-4ebc-802c-91fce9536bcf"}]}

]